dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -24,6 +24,26 @@ _get_cluster_status(){
     echo $(rosa list clusters -o json | jq -r '.[] | select(.name == '\"$1\"') | .status.state')
 }
 
+_wait_for_cluster_ready(){
+    echo "INFO: Waiting about 1 hour for cluster on ready status"
+    # 180 iterations, sleeping 60 seconds, 3 hours of wait
+    for i in `seq 1 180` ; do
+        CLUSTER_STATUS=$(_get_cluster_status $1)
+        if [ ${CLUSTER_STATUS} == "ready" ] ; then
+            sleep 60 # Wait 60 seconds to allow OCM to really finish the cluster installation
```

Comment:
I've also seen that its usually way more than a minute after rosa cli reports cluster as ready, that the cluster is actually ready. So this might need to be bumped.

Reply:
Moved to 2 mintes. anyway, this is just an skell for the function. after cluster is "ready" we should add more checks to verify if cluster is really ready or not. checking just the status returned by rosa is only the beginning of the active waiting, experience will give us the information about what we should check after that